terraform/modules/task-definition/main.tf

```diff
@@ -17,7 +17,8 @@ locals {
       "image" : "840364872350.dkr.ecr.eu-west-1.amazonaws.com/aws-appmesh-envoy:v1.15.0.0-prod",
       "user" : "1337",
       "environment" : [
-        { "name" : "APPMESH_VIRTUAL_NODE_NAME", "value" : "mesh/${var.mesh_name}/virtualNode/${var.service_name}" }
+        { "name" : "APPMESH_VIRTUAL_NODE_NAME", "value" : "mesh/${var.mesh_name}/virtualNode/${var.service_name}" },
+        { "name" : "APPMESH_RESOURCE_ARN", "value" : "mesh/${var.mesh_name}/virtualNode/${var.service_name}" },
```

Comment:
I read `APPMESH_RESOURCE_ARN` is going to replace `APPMESH_VIRTUAL_NODE_NAME`, did you see the same thing? I think we need to add `APPMESH_RESOURCE_ARN` all apps if that's the case. Not necessary for this PR though.

Reply:
Yes, we can do that in another PR, I was confused about whether we needed both but if we need only `APPMESH_RESOURCE_ARN`, then that's fine.